### PR TITLE
Amend title for better clarification

### PR DESCRIPTION
**Pull request by Volker-E**
Amending site title for better clarification and putting technical
note at end.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# WikimediaUI Style Guide
+# Wikimedia Style Guide for the User Interface (WikimediaUI)
```
Comment (prtksxna) [inline README.md]: I am not sure if the //the// is required here. "Wikimedia Style Guide for User Interface (WikimediaUI)" seems fine to me. Same for other files below.

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
 	<link rel="stylesheet" href="css/build/wmui-style-guide.min.css">
-	<title>WikimediaUI Style Guide – provided by Wikimedia Foundation design team</title>
+	<title>Wikimedia Style Guide for the User Interface (WikimediaUI) – provided by Wikimedia Foundation Design Team</title>
```
Comment (prtksxna) [inline index.html]: Same as above.

File: index.html [hunk 3]
```diff
@@ -69,7 +69,7 @@ <h2>How to contribute</h2>
 						<p>You can participate in many ways. The documentation is available in a Git repository with the design assets needed for your project. You can get all of them, change anything and send the changes back.</p>
 
 						<h3>Get the design repository</h3>
-						<p>The <a href="https://github.com/wikimedia/WikimediaUI-Style-Guide/">WikimediaUI Style Guide repository is available to download</a>. It contains this documentation, Sketch and SVG templates and resources. Select the “clone or download” button to get the repository.</p>
+						<p>The <a href="https://github.com/wikimedia/WikimediaUI-Style-Guide/">Wikimedia Style Guide for the User Interface repository is available to download</a>. It contains this documentation, Sketch and SVG templates and resources. Select the “clone or download” button to get the repository.</p>
```
Comment (prtksxna) [inline index.html]: Same as above.

**Comment by Volker-E** [comment]
After conversation last night, we probably will change this once more, as UI doesn't reflect impact on product design. Keeping (WikimediaUI) then for the components and patterns section as addendum…

File: index.html (hunk 2 above)
Comment (Volker-E) [reply index.html]: Done

**Comment by Volker-E** [comment]
This needs to be seen in regards to and preparation of https://phabricator.wikimedia.org/T167164 as well. Amended one more time, we might add “(WikimediaUI)” to the end of components and patterns and also mention it here and there in running text, but leaving it out of the titles.

**Pull request merged**